ghcide/src/Development/IDE/GHC/Util.hs

```diff
@@ -318,4 +314,45 @@ traceAst lbl x
 #endif
             , "file://" ++ htmlDumpFileName]
 
+-- Should in `Development.IDE.GHC.Orphans`,
+-- leave it here to prevent cyclic module dependency
+#if !MIN_VERSION_ghc(8,10,0)
+instance Outputable SDoc where
+  ppr = id
+#endif
 
+-- | Print a GHC value by default `showSDocUnsafe`.
+--
+-- You may prefer `prettyPrint` unless you know what you are doing.
+--
+-- It internal using `unsafeGlobalDynFlags`.
+--
+-- `String` version of `showGhcWithUniques`.
+prettyPrintWithUniques :: Outputable a => a -> String
```

Comment:
I think basically nobody should ever want to do this, right? there's only one use, so maybe inline it there?